### PR TITLE
Adds a Mimicry Equipment Pack to Cargo

### PR DESCRIPTION
**Pull request by Momoberry**
## About the PR 
There's a lot of traders, many are not well known. Geoff sells some mime gear, although just the clothing. I wanted to add this crate to contrast with the clown equipment, obviously bundled with standard mime gear. This contains a classic baguette, a duo of a pack of cigarettes and a zippo lighter, and the alternate mime costume.

## Why's this needed? 
A lot of items are at certain vendors that are generally not common knowledge, and the UI for traders is a little annoying to use, and requires a bank account and ID. The mime costumes are already sold from geoff in the clown hole, but I'd also like cargo to sell a mime bundle, to contrast with the clown bundle. Bundle has (generally) the entire basic mime aesthetic, instead of just the costume like Geoff. 

tl;dr I really like mime, and some other players really like mime. I'll be making mime traitor gear as well, as I'm aiming to have mime become a mainline job-slot instead of a post round in the future 🐌 

## Changelog

```changelog
(u)Momoberry
(+)Added a Mimicry crate to cargo. It comes with a standard issue baguette, full mime attire, a pack of cigarettes and a Zippo.
```

### CODE REVIEW

**Comment by Confusedxoz** [comment]
I can see QM spam buying this to grab cigarettes to sell off to Pianzi. How profitable.
